### PR TITLE
ENG-2070 Add cypress tests for groups

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **raxkaynan**

If they are not already assigned, you can assign the PR to them by writing `/assign @raxkaynan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: test/cypress/integration/groups.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  });
+
+  afterEach(() => {
+    cy.appBuilderLogout();
+  });
+
+  it('Add group', () => {
+    cy.log('should redirect to list with new group after submitting the form');
+    cy.addGroup(groupName);
+    cy.getByTestId('groups-table').should('be.visible');
```
Comment (avdev4j) [inline test/cypress/integration/groups.spec.js:21]: move the id into a constant file

File: test/cypress/support/command/group-command.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  cy.openPageFromMenu(['Users', 'Groups']);
+  cy.getButtonByText('Add').click();
+  cy.getByName('name').type(groupName);
+  cy.getButtonByText('Save').click();
+});
+
+Cypress.Commands.add('deleteGroup', (groupCode) => {
+  cy.openPageFromMenu(['Users', 'Groups']);
+  cy.clickTableActions(groupCode);
+  cy.getByTestId('group-delete-action').filter(':visible').click();
```
Comment (avdev4j) [inline test/cypress/support/command/group-command.js:11]: move the id into a constant file

**Review by avdev4j — CHANGES_REQUESTED** [review]

**Comment by gidesan** [comment]
@raxkaynan anything that blocks you here?

**Review by alepintus — APPROVED** [review]
Looks good

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1036#pullrequestreview-639675349" title="Approved">alepintus</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [alepintus]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**